### PR TITLE
cma 3.2.2 ❄️ 

### PR DESCRIPTION
**Pull request by psteyer**
cma 3.2.2 ❄️ 

**Destination channel:** Snowflake

### Links

- [PKG-4426](https://anaconda.atlassian.net/browse/PKG-4426) 
- [Upstream repository](https://github.com/CMA-ES/pycma)
- [Upstream changelog/diff](https://github.com/CMA-ES/pycma/compare/r3.2.2...r3.3.0)

### Explanation of changes:

- bump version
- updated SHA
- added script flags
- added skip for py3.6
- removed python specific pinnings
- added host requirements
- added test commands and requires
- added dev_url
- updated doc_url

[PKG-4426]: https://anaconda.atlassian.net/browse/PKG-4426?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by psteyer** [comment]
With this request being building an older version of `cma`, do I need to do anything special?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,40 +1,47 @@
...
-  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: True # [py<36]
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
     - numpy
```
Comment (lorepirri) [inline recipe/meta.yaml]: Does it build without `numpy` in the `host`? I see it only in `install_requires`

```suggestion
```

**Review by lorepirri — APPROVED** [review]
LGTM,

remove `numpy` from the `host` if not needed,

you could add a test, see https://inspector.pypi.io/project/cma/3.2.2/packages/91/3c/85f3a10db637d2c8776a4154d593a7fd820be8d900fde5514df45ebac466/cma-3.2.2.tar.gz/cma-3.2.2/cma/test.py

**Review by cbouss — APPROVED** [review]
LGTM, if @lorepirri's comment is addressed.

**Comment by lorepirri** [comment]
> With this request being building an older version of `cma`, do I need to do anything special?

it is an old version but it was never in our channels, therefore for us it is the first version, no need to do anything special

**Review by lorepirri — APPROVED** [review]
LGTM, well done, thanks for adding the tests!

**Pull request merged**